marketplace/commercetools-products/src/ct-product-query.graphql

```diff
@@ -0,0 +1,27 @@
+query productsList($skus: [String!]) {
```

Comment:
Is this the only query? Can we use the REST API instead? I don't see too much value in bundling Apollo if we can get the same data with a single `fetch` call.